x-pack/plugins/osquery/public/results/attachments/external_reference.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ExternalReferenceAttachmentType } from '@kbn/cases-plugin/public/client/attachment_framework/types';
+import { lazy } from 'react';
+// import { EuiButtonIcon } from '@elastic/eui';
+
+const AttachmentContentLazy = lazy(() => import('./external_references_content'));
+
+// TODO waiting for Metadata to add "add to timeline" in here
+// const AttachmentActions: React.FC = () => (
+//   <EuiButtonIcon
+//     data-test-subj="test-attachment-action"
+//     onClick={() => {}}
+//     iconType="arrowRight"
+//     aria-label="See attachment"
+//   />
+// );
+
+export const getExternalReferenceAttachmentRegular: () => ExternalReferenceAttachmentType = () => ({
+  id: 'osquery',
+  icon: 'logoOsquery',
+  displayName: 'Osquery',
+  getAttachmentViewObject: () => ({
+    type: 'regular',
+    event: 'attached Osquery results',
+    timelineIcon: 'logoOsquery',
```

Comment:
Is this a valid icon? I do not see it here: https://elastic.github.io/eui/#/display/icons

Reply:
No it doesnt seem to be. But none icons work if I am not mistaken, always gets this 
![Zrzut ekranu 2022-09-14 o 15 56 12](https://user-images.githubusercontent.com/16632552/190173901-c2b381a3-54d7-4f44-9e97-9d999efc1547.png)

Reply:
A valid icon should work. If you always get this it means you are using a nonvalid icon.

Reply:
Ok this is changed - wrong props name.